src/main/groovy/com/ullink/NuGetPlugin.groovy

```diff
@@ -27,6 +27,11 @@ class NuGetPlugin implements Plugin<Project> {
             group = BasePlugin.UPLOAD_GROUP
             description = 'Pushes the NuGet package to the configured server url.'
         }
+
+        project.task('nugetSources', type: NuGetSources) {
+            group = BasePlugin.UPLOAD_GROUP
```

Comment:
probably this operation shouldn't be part of the upload group, you can probably remove the group entirely if there's no fitting/existing one

Reply:
Ok, I have removed the group.